src/scenes/Languages/LanguageForm/LanguageForm.tsx

```diff
@@ -191,6 +191,17 @@ export const LanguageForm = <T extends any>({
                       </Grid>
                     )}
                   </SecuredField>
+                  <SecuredField obj={language} name="hasExternalFirstScripture">
+                    {(props) => (
+                      <Grid item xs={12} sm={6}>
+                        <CheckboxField
+                          label="Was this language's first scripture produced outside of CORD?"
+                          margin="none"
```

Comment:
Can you add the `keepHelperTextSpacing` flag here? This is an unusual case where we are expecting the server to error on this field if it's not valid based on the rules.